CHANGELOG.md

```diff
@@ -27,7 +27,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Update version from 0.x.x to 7.0.0 to prep for the GA release
 - Update help screens according to [these guidelines](https://ljfz3b.axshare.com/#id=yokln4&p=conjur_main_help&g=1).
   [cyberark/conjur-api-python3#92](https://github.com/cyberark/conjur-api-python3/issues/92)
-- Directory Structure Changed [dir changes](https://github.com/cyberark/conjur-api-python3/blob/design/general-refactorings/design/general_refactorings.md#organize-directory-structure)
+- Directory structure has been changed. See [design document](https://github.com/cyberark/conjur-api-python3/blob/design/general-refactorings/design/general_refactorings.md#organize-directory-structure) for more details.
```

Comment:
```
Users who import the SDK should change their import statement from `conjur.client import Client` to  `from conjur.api.client import Client`[cyberark/conjur-api-python3#121](https://github.com/cyberark/conjur-api-python3/issues/121)
```

Reply:
@orenbm  👀 
We are changing project directory which means this is a breaking change for those that import the Client in their projects. How is the recommended changelog entry?

Reply:
well, users would know that the API has been broken by the version as its MAJOR should change.
However, iirc this is version 0 so we don't need to do that if we don't want to. I think that on version 0 users should assume that the API may break.

Maybe we can write that this change breaks the API?

Reply:
we did something similar here: https://github.com/cyberark/conjur-authn-k8s-client/blob/master/CHANGELOG.md#changed

Reply:
Unless we post it somewhere, I don't think users would assume breaking changes could regularly occur without a major number change. If this change is breaking, it should absolutely get a major version bump on the next release.

Reply:
Fixed

Reply:
It will be a part of the coming soon CLI release so no need for special version bump for it.